htlcswitch/interceptable_switch.go

```diff
@@ -173,6 +179,11 @@ type InterceptableSwitchConfig struct {
 	// RequireInterceptor indicates whether processing should block if no
 	// interceptor is connected.
 	RequireInterceptor bool
+
+	// SignAliasUpdate is used when an intercepting application includes an
+	// unsigned channel update to be signed by us.
+	SignChannelUpdate func(u *lnwire.ChannelUpdate) (*ecdsa.Signature,
```

Comment:
Godoc mismatch, but see above, not sure we want to allow this.

Reply:
Fixed godoc. Discussion can continue in https://github.com/lightningnetwork/lnd/pull/7067#discussion_r1201447182